controller/api/urls.py

```diff
@@ -13,59 +17,58 @@
-    url(r'^apps/(?P<id>{})/containers/(?P<type>[-_\w.]+)/restart/?'.format(settings.APP_URL_REGEX),
+    url(r"^apps/(?P<id>{})/containers/(?P<type>[-_\w.]+)/restart/?".format(settings.APP_URL_REGEX),
         views.ContainerViewSet.as_view({'post': 'restart'})),
-    url(r'^apps/(?P<id>{})/containers/(?P<type>[-_\w]+)/(?P<num>[-_\w]+)/restart/?'.format(
+    url(r"^apps/(?P<id>{})/containers/(?P<type>[-_\w]+)/(?P<num>[-_\w]+)/restart/?".format(
         settings.APP_URL_REGEX),
         views.ContainerViewSet.as_view({'post': 'restart'})),
-    url(r'^apps/(?P<id>{})/containers/(?P<type>[-_\w]+)/(?P<num>[-_\w]+)/?'.format(
+    url(r"^apps/(?P<id>{})/containers/(?P<type>[-_\w]+)/(?P<num>[-_\w]+)/?".format(
         settings.APP_URL_REGEX),
         views.ContainerViewSet.as_view({'get': 'retrieve'})),
-    url(r'^apps/(?P<id>{})/containers/(?P<type>[-_\w.]+)/?'.format(settings.APP_URL_REGEX),
+    url(r"^apps/(?P<id>{})/containers/(?P<type>[-_\w.]+)/?".format(settings.APP_URL_REGEX),
         views.ContainerViewSet.as_view({'get': 'list'})),
-    url(r'^apps/(?P<id>{})/containers/?'.format(settings.APP_URL_REGEX),
+    url(r"^apps/(?P<id>{})/containers/?".format(settings.APP_URL_REGEX),
         views.ContainerViewSet.as_view({'get': 'list'})),
     # application domains
-    url(r'^apps/(?P<id>{})/domains/(?P<domain>[-\._\w]+)/?'.format(settings.APP_URL_REGEX),
+    url(r"^apps/(?P<id>{})/domains/(?P<domain>[-\._\w]+)/?".format(settings.APP_URL_REGEX),
         views.DomainViewSet.as_view({'delete': 'destroy'})),
-    url(r'^apps/(?P<id>{})/domains/?'.format(settings.APP_URL_REGEX),
+    url(r"^apps/(?P<id>{})/domains/?".format(settings.APP_URL_REGEX),
         views.DomainViewSet.as_view({'post': 'create', 'get': 'list'})),
     # application actions
-    url(r'^apps/(?P<id>{})/scale/?'.format(settings.APP_URL_REGEX),
+    url(r"^apps/(?P<id>{})/scale/?".format(settings.APP_URL_REGEX),
         views.AppViewSet.as_view({'post': 'scale'})),
-    url(r'^apps/(?P<id>{})/logs/?'.format(settings.APP_URL_REGEX),
+    url(r"^apps/(?P<id>{})/logs/?".format(settings.APP_URL_REGEX),
         views.AppViewSet.as_view({'get': 'logs'})),
-    url(r'^apps/(?P<id>{})/run/?'.format(settings.APP_URL_REGEX),
+    url(r"^apps/(?P<id>{})/run/?".format(settings.APP_URL_REGEX),
         views.AppViewSet.as_view({'post': 'run'})),
     # apps sharing
-    url(r'^apps/(?P<id>{})/perms/(?P<username>[-_\w]+)/?'.format(settings.APP_URL_REGEX),
+    url(r"^apps/(?P<id>{})/perms/(?P<username>[-_\w]+)/?".format(settings.APP_URL_REGEX),
         views.AppPermsViewSet.as_view({'delete': 'destroy'})),
-    url(r'^apps/(?P<id>{})/perms/?'.format(settings.APP_URL_REGEX),
+    url(r"^apps/(?P<id>{})/perms/?".format(settings.APP_URL_REGEX),
         views.AppPermsViewSet.as_view({'get': 'list', 'post': 'create'})),
     # apps base endpoint
-    url(r'^apps/(?P<id>{})/?'.format(settings.APP_URL_REGEX),
+    url(r"^apps/(?P<id>{})/?".format(settings.APP_URL_REGEX),
```

Comment:
Python doesn't care which quotes you use, but previously a couple of us had maintained lots of CoffeeScript code alongside Python. It's not consistent in the codebase, but I have a preference for Ruby-style quoting even in Python code to indicate strings which are used for templating.

Obviously it doesn't matter either way; we used to think it was a bit helpful to indicate intent.